charmtools/build/fetchers.py

```diff
@@ -112,10 +113,11 @@ def fetch(self, dir_):
 
 
 class LayerFetcher(InterfaceFetcher):
-    INTERFACE_DOMAIN = "http://interfaces.juju.solutions"
+    INTERFACE_DOMAIN = "https://raw.githubusercontent.com/" \
```

Comment:
Would it be worth moving this from away from raw to a github site? raw access is generally discouraged.